includes/Core/User_Input/User_Input.php

```diff
@@ -97,7 +98,8 @@ public function __construct(
 		$this->user_specific_answers = new User_Specific_Answers( $this->user_options );
 		$this->rest_controller       = new REST_User_Input_Controller(
 			$this,
-			$survey_queue ?: new Survey_Queue( $this->user_options )
+			$survey_queue ?: new Survey_Queue( $this->user_options ),
+			new Is_Key_Metrics_Setup_Complete( $options ?: new Options( $context ) )
```

Comment:
Thinking about this again, let's not couple user input with key metrics directly here.

Instead, I think we can listen for the site-specific answers being saved/updated (`googlesitekit_user_input_settings`) and use that to conditionally set the flag in `Key_Metrics`. That should be a relatively small change to make and keep the logic for this under the KM realm of the codebase.

Reply:
Leaving as defined in IB for now.